src/main/java/sirius/biz/codelists/CodeListExportJob.java

```diff
@@ -0,0 +1,135 @@
+                             ImportDictionary dictionary,
+                             List<String> defaultMapping,
+                             ProcessContext process,
+                             String factoryName,
+                             Optional<String> languageParameter,
+                             CodeList codeList) {
+        super(templateFileParameter,
+              destinationParameter,
+              fileTypeParameter,
+              type,
+              dictionary,
+              defaultMapping,
+              process,
+              factoryName);
+        this.codeList = codeList;
+        this.languageParameter = languageParameter;
+        languageParameter.ifPresent(lang -> {
+            this.languagePrefix = lang + "_";
+            this.defaultMapping.add(languagePrefix + CLE_VALUE);
+            this.defaultMapping.add(languagePrefix + CLE_ADDITIONAL_VALUE);
+            this.defaultMapping.add(languagePrefix + CLE_DESCRIPTION);
+
+            this.dictionary.addField(FieldDefinition.stringField(languagePrefix + CLE_VALUE)
+                                                    .withLabel(languagePrefix.toUpperCase() + NLS.get(
+                                                            "CodeListEntryData.value",
+                                                            lang))
+                                                    .addAlias(languagePrefix + CLE_VALUE));
+            this.dictionary.addField(FieldDefinition.stringField(languagePrefix + CLE_ADDITIONAL_VALUE)
+                                                    .withLabel(languagePrefix.toUpperCase() + NLS.get(
+                                                            "CodeListEntryData.additionalValue",
+                                                            lang))
+                                                    .addAlias(languagePrefix + CLE_ADDITIONAL_VALUE));
+            this.dictionary.addField(FieldDefinition.stringField(languagePrefix + CLE_DESCRIPTION)
+                                                    .withLabel(languagePrefix.toUpperCase() + NLS.get(
+                                                            "Model.description",
+                                                            lang))
+                                                    .addAlias(languagePrefix + CLE_DESCRIPTION));
+        });
+    }
+
+    @Override
+    public void execute() throws Exception {
+        process.addLink(new ProcessLink().withLabel("$CodeList").withUri("/code-list/" + codeList.getIdAsString()));
+        process.updateTitle(Strings.apply("%s - %s", process.getTitle(), codeList.getCodeListData().getName()));
+        super.execute();
+    }
+
+    @Nullable
+    @Override
+    protected Function<E, Object> customFieldExtractor(String field) {
+        if (!field.startsWith(languagePrefix) || !languageParameter.isPresent()) {
+            return null;
+        }
+
+        String realField = field.replaceFirst(languagePrefix, "");
```

Comment:
doesn't this belong in the import handler?

Reply:
This also prevents unnecessary code duplication (see below)

Reply:
To prevent code duplication, it has been agreed to keep the code in this class.

Reply:
EntityImportHandlerExtender could hande both - see PerformanceDataXXX or PermissionDataXXX

Reply:
That is a valid suggestion, but after looking into it not feasible at the moment. We need access to parameters from the jobs to build things like exportable mappings dynamically, which is currently not possible with HandlerExtenders. 
I suggest we do a larger rework similar to the universal import as a follow-up